src/utils/email/email.js

```diff
@@ -0,0 +1,10 @@
+import sgMail from '@sendgrid/mail';
+
+sgMail.setApiKey(process.env.SENDGRID_API_KEY);
+
+const send = (params) => sgMail.send({
+  ...params,
+  from: process.env.SENDGRID_EMAIL_FROM,
+});
```

Comment:
```suggestion
```suggestion
const { SENDGRID_EMAIL_FROM, SENDGRID_API_KEY } = process.env;

sgMail.setApiKey(SENDGRID_API_KEY);

const send = (params) => sgMail.send({
  ...params,
  from: SENDGRID_EMAIL_FROM,
});
```